alphabet/alphabet_contract.go

```diff
@@ -34,13 +33,21 @@ func OnNEP17Payment(from interop.Hash160, amount int, data interface{}) {
 
 func _deploy(data interface{}, isUpdate bool) {
 	ctx := storage.GetContext()
+
+	//TODO(@acid-ant): #9 remove notaryDisabled from args in future version
+	if data.([]interface{})[0].(bool) {
+		panic(common.PanicMsgForNotaryDisabledEnv)
+	}
```

Comment:
It looks complex, can we move `args := data.(struct` cast here (it is already done below) and check `args.notaryDisabled` field?

Reply:
Ok, refactored.